data/data/bootstrap/baremetal/files/etc/NetworkManager/conf.d/99-baremetal.conf

```diff
@@ -1,3 +1,14 @@
 [connection]
 ipv6.dhcp-duid=ll
 ipv6.dhcp-iaid=mac
+
+{{if .PlatformData.BareMetal.ExternalStaticIP}}
```

Comment:
I think this needs to go into a new file e.g `/etc/NetworkManager/system-connections/ens3.nmconnection`, not the global `99-baremetal-conf`?

Reply:
OK. Can we hardcode `ens3`?

Reply:
That's a good point - no I don't think we can, specifically the name differs for ARM, so we'll need to use the MAC address provided via https://github.com/openshift/installer/pull/5698 (if specified) then default to the name when no MAC is specified 

@jeffdyoung would you agree?